### ISSUE TITLE
bug: no keyboard navigation for tippy menus

### ISSUE THREAD

**Issue by nassdonald** [EmerisHQ/demeris#1077]
**Describe the bug**
We're currently using a Vue plugin called Tippy for tooltips. This plugin has also been used to create custom dropdown menus, but it's not entirely satisfying all usability requirements with the current implementations - e.g. See staking section > validator list item > actions menu.
There are two issues at play here, one is a bug, the other is a lack of expected functionality (feature):
1. (Feature) it's currently not possible to navigate through the options in a dropdown menu.
2. (Bug) it's currently possible to show multiple menus at once
![image](https://user-images.githubusercontent.com/2961244/154836499-7e9fab62-c9ad-473a-9925-28f6b515e052.png)

**To Reproduce**
1. Go to asset detail view > Staking section
2. Navigate to validator actions menu trigger (ellipsis icon) with the keyboard
3. Trigger a button click with return key
4. Menu opens, but menu doesn't have focus and options aren't navigable with keyboard (e.g. up/down keys or tab)
5. Navigate to the next actions menu trigger in the list (tab key) - previous menu stays open
6. As step 3
7. As step 4 - previous menu and new menu both open.

**Expected behavior**
It should be possible to navigate through the items in a dropdown menu with keyboard navigation, in a non-modal state. This expected behavior can be seen in typical dropdown menus in other applications - for example, try the "+" menu here in GitHub itself.
![image](https://user-images.githubusercontent.com/2961244/154836756-ff2e2d08-2469-4f77-9eef-4b24e4b167d5.png)

It should not be possible to open multiple menus at once.

There are likely 

**Additional context**
It may be necessary to seek an alternative to Tippy for dropdown menus - either a custom built component (likely a lot of work) or another plugin - if Tippy doesn't provide or allow this functionality elegantly.
